statsmodels/tsa/arima_model.py

```diff
@@ -858,13 +876,20 @@ def fit(self, start_params=None, trend='c', method="css-mle",
             If True, all output from solver will be available in
             the Results object's mle_retvals attribute.  Output is dependent
             on the solver.  See Notes for more information.
-        disp : bool, optional
+        disp : int, optional
```

Comment:
My guess is that this depends on the scipy optimizer, but lbfgsb has numeric options.